docs/user-guide.md

```diff
@@ -780,24 +794,24 @@ need a different value. The parameter values that the toolkit uses are echoed
 during execution so you can confirm the configuration.
 
 The complete list of parameters and their values are provided in the [Parameters
-section](#heading=h.m1e7sxtn4kzm).
+section](#parametersn).
```

Comment:
I think there's an extraneous "n" and the end of "#parameters"

Reply:
And by the way, great catch on these incorrect internal links. They are leftover from the Google doc. I should have caught them.

Reply:
Sentence removed, thanks.